tests/src/Kernel/Framework/UploadFileServiceTest.php

```diff
@@ -15,11 +15,6 @@
  */
 class UploadFileServiceTest extends GraphQLTestBase {
 
-  /**
-   * {@inheritdoc}
-   */
-  protected static $modules = ['file', 'graphql_file_validate'];
```

Comment:
@apathak18 Why we removed it here ? It's better we add reason here, so maintainers are aware of on why this change was made.

Reply:
Parent class graphqlTestBase already have file module.
And graphql_file_validate is only testing for hook_file_validate which is deprecated in D10.2 and we are using file.validator service as a replacement.